core/api/src/main/java/com/blazebit/persistence/spi/DbmsDialect.java

```diff
@@ -205,6 +206,22 @@
     public boolean supportsJoinsInRecursiveCte();
 
     /**
+     * Returns true if the dbms supports row value constructor syntax, false otherwise.
+     *
+     * @return Whether row value constructor syntax is supported by the dbms
+     * @since 1.2.0
+     */
+    public boolean supportsRowValueConstructor();
+
+    /**
+     * Returns true if the dbms supports <, <=, >, >=, =, <> comparison operations for row values.
```

Comment:
The documentation is misleading. If the DBMS supports the syntax it has to at least support equality checks i.e. operators `=` and `<>`. If you have no idea for what you might use the `supportsRowValueConstructor` I suggest you remove it and just keep this method.
Also note that there is a nice overview of the support in various DBMS here: https://blog.jooq.org/2012/12/26/row-value-expressions-and-the-between-predicate/